packages/zowe-explorer/README.md

```diff
@@ -319,16 +273,16 @@ As a result, VS Code extensions like Zowe Explorer, Zowe Explorer CICS Extension
 ### Edit and upload a data set member
 
 1. Navigate to the **Side Bar**.
-2. Open the **DATA SETS** bar.
-3. Open a profile.
-4. Select the data set member you want to edit.
+1. Open the **DATA SETS** bar.
+1. Open a profile.
+1. Select the data set member you want to edit.
 
    **Note:** To view the members of a data set, click the data to expand the tree.
 
    The data set member is displayed in the text editor window of VS Code.
 
-5. Edit the document.
-6. Navigate back to the data set member in the explorer tree, and press `Ctrl`+`S` or `Command`+`S` (OSx) to upload the member.
+1. Edit the document.
+1. Navigate back to the data set member in the explorer tree, and press `Ctrl`+`S` or `Command`+`S` (OSx) to upload the member.
```

Comment:
We should probably update `Command` + `S` (OSx) to `Command` + `S` (macOS) since they renamed to macOS